src/main/java/com/datatorrent/netlet/util/VarInt.java

```diff
@@ -59,6 +61,18 @@ public static int write(int value, byte[] buffer, int offset, int size)
     return i;
   }
 
+  public static int write(int value, ByteBuffer buffer)
```

Comment:
Can the buffer be native/direct? If not, so that the same logic is not repeated, can't this be a call to write(value, buffer.array(), buffer.position()). Also, I think the implementation below is more compact and can replace the original implementation.